Source/Hurl.BrowserSelector/Windows/MainWindow.xaml

```diff
@@ -2,174 +2,141 @@
     x:Class="Hurl.BrowserSelector.Windows.MainWindow"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:controls="clr-namespace:Hurl.BrowserSelector.Controls"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:local="clr-namespace:Hurl.BrowserSelector.Windows"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     xmlns:tray="http://schemas.lepo.co/wpfui/2022/xaml/tray"
-    xmlns:vs="clr-namespace:Hurl.BrowserSelector.Controls"
     xmlns:wpfui="http://schemas.lepo.co/wpfui/2022/xaml"
-    Title="Hurl - Select a Browser"
+    Title="Select Browser - Hurl"
     MinWidth="410"
     MinHeight="220"
-    MaxWidth="560"
+    MaxWidth="720"
     MaxHeight="450"
     Deactivated="Window_Deactivated"
     ExtendsContentIntoTitleBar="True"
+    Icon="/Windows/internet.ico"
```

Comment:
@Jay-o-Way How does adding Icon help here, since Icon is already being shown for Windows ?